docs/source/examples/README.rst

```diff
@@ -7,9 +7,8 @@ Examples
 Here is a collection of example Jupyter notebooks.
 Each notebook can be downloaded to run on your own device.
 
-
 .. toctree::
    :maxdepth: 1
+   :glob:
 
-   resp_single
-   resp_multi
+   *example*
```

Comment:
@hannahbrucemacdonald is this what you meant by adding files in automatically instead of updating the README? Ideally the notebooks can be run out of the box, they're just hosted under docs/sources/examples because of limitations from sphinx as to where files are allowed to be. I added a symlink to the top level but unfortunately GitHub renders it as a text file instead of a helpful link.

Reply:
I believe that it has to do with the main README file. The specifics of of how the sphinx rst listing is update is up to the project's developers.